deployment/manifests/default.pp

```diff
@@ -33,11 +35,18 @@
 
 # Django-docker-engine needs a place for ephemeral data.
 # In production, this is a separate EBS mount, so we don't need to create it locally.
-file { '/data':
-  ensure => 'directory',
+file { $::data_dir:
+  ensure => directory,
   owner  => $app_user,
   group  => $app_group
 }
+->
+file { $::django_docker_engine_data_dir:
```

Comment:
This violates DRY because this exact same resource is declared for both AWS and Vagrant manifests.

Reply:
@hackdna Okay, do you have any ideas as to how I should factor out the similarities? Its great that you are pointing out my faults, but without research on my end I may not be able to make these changes as quickly as you could, and our current release is highly dependent on this fix.

Reply:
Sure, classes are a great way to keep the code DRY and re-usable, for example: https://github.com/refinery-platform/refinery-platform/pull/2175/files#diff-cef6f90f8edfbbb045c18b4f98d5e4a6

Reply:
@hackdna Cool this is great.

I'm just thinking that in the context of the `1.6.0` release we should only be addressing bugs and I'd like to introduce the minimum amount of changes possible to fix #2184.

If I introduced changes like you've done in #2175 wouldn't we theoretically have to re-test everything that we've already done?

Would you considering pinning down working functionality with this PR since #2175 seems like a great base to resolve the DRY-ness?

Reply:
OK. I've already merged changes into PR #1275 from develop that happened after creating the release-1.6.0 branch, so at this point PR #1275 should only be merged back into develop.

Going forward, it would be great to figure out a way to fix bugs without introducing even more technical debt.

Reply:
@hackdna Thats fine, I was more so wondering if we could address the DRYness here in #1257 since that seems like a perfect place to do so , and it wouldn't require us to test everything for `1.6.0` over again.

Reply:
Sure, as long as you don't mind the increase in technical debt.